doc/introduction/importing_circuits.rst

```diff
@@ -0,0 +1,267 @@
+execute the circuit remotely using a supported hardware provider.
+
+Qiskit
+~~~~~~
+
+To import a quantum circuit from Qiskit, you must first install the `PennyLane-Qiskit
+<https://docs.pennylane.ai/projects/qiskit/en/stable/>`__ plugin. This can be done by running:
+
+.. code-block::
+
+    pip install pennylane-qiskit
+
+Now, suppose we define a Qiskit `QuantumCircuit
+<https://docs.quantum.ibm.com/api/qiskit/qiskit.circuit.QuantumCircuit>`__ as follows:
+
+.. code-block:: python
+
+    from qiskit import QuantumCircuit
+
+    qk_circuit = QuantumCircuit(2, 1)
+    qk_circuit.h(0)
+    qk_circuit.cx(0, 1)
+    qk_circuit.measure_all()
+
+We can convert the ``QuantumCircuit`` into a PennyLane quantum function using:
+
+.. code-block:: python
+
+    import pennylane as qml
+
+    pl_template_from_qk = qml.from_qiskit(qk_circuit)
+
+Above, the :func:`from_qiskit` function converts a ``QuantumCircuit`` into a PennyLane template.
+This template can then be called from inside a QNode to generate a PennyLane circuit:
+
+.. code-block:: python
+
+    @qml.qnode(qml.device("default.qubit"))
+    def pl_circuit_from_qk():
+        pl_template_from_qk(wires=[0, 1])
+        return qml.expval(qml.Y(0)), qml.expval(qml.Z(1))
+
+.. note::
+
+    Alternatively, the QNode can be instantiated directly from the Qiskit circuit:
+
+    .. code-block:: python
+
+        measurements = [qml.expval(qml.Y(0)), qml.var(qml.Z(1))]
+        pl_template_from_qk = qml.from_qiskit(qk_circuit, measurements=measurements)
+        pl_circuit_from_qk = qml.QNode(pl_template_from_qk, qml.device("default.qubit"))
+
+
+    Here, the ``measurements`` argument overrides the terminal measurements in the Qiskit circuit.
+    See the :func:`from_qiskit` documentation for more details.
+
+The resulting PennyLane circuit can be executed directly:
+
+>>> pl_circuit_from_qk()
+[tensor(0., requires_grad=True), tensor(1., requires_grad=True)]
+
+It can also be visualized using PennyLane's :func:`~pennylane.draw` utility:
+
+>>> print(qml.draw(pl_circuit_from_qk)())
+0: ──H─╭●─╭||─┤  <Y>
+1: ────╰X─╰||─┤  Var[Z]
+
+OpenQASM
+~~~~~~~~
+
+An equivalent quantum circuit can be expressed in OpenQASM 2.0 as follows:
+
+.. code-block:: python
+
+    oq_circuit = (
+        """
+        OPENQASM 2.0;
+        include "qelib1.inc";
+        qreg q[2];
+        creg c[2];
+
+        h q[0];
+        cx q[0], q[1];
+
+        measure q -> c;
+        """
+    )
+
+We can import this circuit into PennyLane using the PennyLane-Qiskit plugin once more:
+
+.. code-block:: python
+
+    import pennylane as qml
+
+    pl_template_from_oq = qml.from_qasm(oq_circuit)
+
+    @qml.qnode(qml.device("default.qubit"))
+    def pl_circuit_from_oq():
+        pl_template_from_oq(wires=[0, 1])
+        return qml.expval(qml.Y(0)), qml.var(qml.Z(1))
+
+The result is as follows:
+
+>>> print(qml.draw(pl_circuit_from_oq)())
+0: ──H─╭●──┤↗├─┤  <Y>
+1: ────╰X──┤↗├─┤  Var[Z]
```

Comment:
I don't love that the measurements are included, see post in Slack. Not a problem with the QuickStart - if anything, evidence that the QuickStart demonstrates the existing behaviour well. But I don't think this is what we want, unfortunately.

Reply:
That is true although perhaps that makes this a bad candidate circuit for the quickstart. Would it be an improvement if we just change the circuits to measure w.r.t. `qml.Z` instead?

Reply:
What happens if we remove the measure from the QASM script?

Reply:
I just think most QASM scripts are device instructions and include measure at the end, but I might be wrong. We could just remove it from the QASM script, so we aren't emphasizing a behaviour (automatically overwriting measurements) that we will likely change soon, and then think more about this when we update the QASM conversion functionality in the future.

Reply:
@lillian542 @trbromley So is the consensus to remove the measurement from the end of the QASM script?

Reply:
Yes please